ferrocene/packages.toml

```diff
@@ -91,9 +91,9 @@ targets = [
   "x86_64-pc-nto-qnx710",
 ]
 
-[[groups.cross-compilation.packages]]
```

Comment:
Did you mean to delete this???

Reply:
Yes. With the old one, it'd add the package to the cross-compilation group, not to the qnx group.